src/src/main.java

```diff
@@ -0,0 +1,21 @@
+public class main {
+    public static void main(String[] args){
+
+        int[] zahlen={4, 8, 3, 9};
+
+        for (int i = 0; i <zahlen.length ; i++) {
+            int sortieren = zahlen[i];
+            int j= i;
+
+            while (j>0 && sortieren<zahlen[j - 1]){
+                zahlen[j]=zahlen[j - 1];
+                j--;
+            }
+            zahlen[j]= sortieren;
+        }
+        System.out.println(zahlen[0]);
```

Comment:
👍 du kannst `Arrays.toString(zahlen)` verwenden um das ganze array in einem hübschen string auszugeben